app/views/admin/users/_form.html.erb

```diff
@@ -22,5 +22,8 @@
 <div class="row">
   <div class="col-md-12">
     <%= link_to icon_tag('glyphicon-chevron-left') + ' Back'.html_safe, admin_users_path, class: "btn btn-default" %>
+    <% if @user.id %>
```

Comment:
You could use `@user.persisted?`, also the following line needs to be indented by two spaces.